inc/Engine/Optimization/RUCSS/Controller/UsedCSS.php

```diff
@@ -807,6 +819,14 @@ private function add_used_fonts_preload( string $html, string $used_css ): strin
 				continue;
 			}
 
+			// Combine the array elements into a single string with | as a separator.
+			$exclude_fonts_preload_pattern = implode( '|', array_map( 'preg_quote', $exclude_fonts_preload ) );
+
+			// Check if the font URL matches any part of the exclude_fonts_preload array.
+			if ( preg_match( '/' . $exclude_fonts_preload_pattern . '/i', $font_url ) ) {
```

Comment:
You will need to either:
- change the delimiter, since there might be `/` in the patterns, and `preg_quote()` doesn't escape them by default
- escape `/` with the extra parameter of `preg_quote()`

Reply:
What do you think of something like this:
```php
			// Combine the array elements into a single string with | as a separator and returning a pattern.
            $exclude_fonts_preload_pattern = implode( '|', array_map( function($item) {
                return preg_quote($item, '/');
            }, $exclude_fonts_preload ) );

			// Check if the font URL matches any part of the exclude_fonts_preload array.
			if ( preg_match( '/' . $exclude_fonts_preload_pattern . '/i', $font_url ) ) {
				continue; // Skip this iteration as the font URL is in the exclusion list.
			}

```

Reply:
this works

Reply:
Change done :)